### PR TITLE
formatted app page, added header & home button (base.html)

### PR DESCRIPTION
**Pull request by smwatts**
**1. Added a header**
*Used base.html to allow all pages to inherit the same header*

I didn't add a footer - couldn't find any nice ones from the Waterloo website, we can discuss this and see if its worth adding.

Header looks like this:
screen shot 2018-02-10 at 12 51 41 pm

**2. Application: Basic Information**
The application page "basic info" section has been formatted:
screen shot 2018-02-10 at 12 14 27 pm

**3. Home/Logout button**
*Also using base.html, ensured all AC pages pass the variable 'AC': AC in views*
I added a home & logout button based on the base.html for all pages the AC enters off the home page:
screen shot 2018-02-11 at 11 50 17 am

Let me know what you think!

### CODE REVIEW

**Comment by j2kan** [comment]
remove all the ds store and your script please :)

**Comment by smwatts** [comment]
@j2kan fixed!!

File: taworks/taform/static/taform/css/style.css [hunk 1]
```diff
@@ -102,3 +102,96 @@ textarea {
...
+    text-align: center;
+    background-color: #A9A9A9;
+    border-radius: 50%;
+    width: 12px;
+    height: 12px;
+    line-height: 12px;
+    cursor: default;
+}
+
+.help-tip:before{
```
Comment (j2kan) [inline taworks/taform/static/taform/css/style.css:123]: whats the difference between 

.help-tip

and

.help tip p?

which one gets used when you call class=help-tip?

**Comment by j2kan** [comment]
I think it might be helpful to do the following.

https://www.quora.com/How-do-I-define-Django-site-wide-templates-like-the-headers-and-footers

make a base page and have all the other pages extend it. This will be easier towork with when we want to add more to the header such as university of waterloo mgmt.. and some type of footer.

**Comment by smwatts** [comment]
Thanks @j2kan Ill look at that!

File: taworks/taform/static/taform/css/style.css (hunk 1 above)
Comment (smwatts) [reply taworks/taform/static/taform/css/style.css:123]: so help tip p is what formats the actual wording within the "?" button. The help tip is the formatting for the "?" button. Does that make sense?

**Comment by smwatts** [comment]
@j2kan -> changed PR so the page formatting now referencing base.html, great suggestion!

File: README.md [hunk 2]
```diff
@@ -26,6 +26,6 @@
    * Enter `python manage.py makemigrations`
    * Enter `python manage.py migrate`
 3. Run static files
-   * Enter `python manage.py collectstatics`
+   * Enter `python manage.py collectstatic`
```
Comment (j2kan) [inline README.md:29]: thanks for updating that

File: taworks/taform/static/taform/css/style.css (hunk 1 above)
Comment (j2kan) [reply taworks/taform/static/taform/css/style.css:123]: yep crystal

File: taworks/taform/templates/taform/base.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+<header id = "banner"></header>
+<head>
+</head>
+<body>
+{% if AC %}
+<table align="right">
+	<tr>
+		<td>
+			<form>
+		        <button class="AC_button", formaction="{% url 'home' %}">Home</button>
```
Comment (j2kan) [inline taworks/taform/templates/taform/base.html]: can we align the code spaces up on this page?

File: taworks/taform/views.py [hunk 4]
```diff
@@ -500,3 +510,9 @@ def determine_status(df):
         status = df_max.iloc[0]['status']
     app_status = 'Open' if status else 'Closed' 
     return (status_date, status, app_status)
+
+def authenticated(request):
```
Comment (j2kan) [inline taworks/taform/views.py:525]: nice job moving this out and creating a method since it's being used multiple times now.

**Review by j2kan — COMMENTED** [review]
Thanks for taking the time to use the base template setup.

Just have to make the base.html page code tabs/spaces/style better and it's LGTM.

File: taworks/taform/templates/taform/base.html (hunk 3 above)
Comment (smwatts) [reply taworks/taform/templates/taform/base.html]: I think its just Github being weird with the display:
screen shot 2018-02-11 at 3 40 52 pm

**Review by j2kan — DISMISSED** [review]
lgtm

**Review by amyleblond — DISMISSED** [review]
it does look pretty :) good to go

**Review by amyleblond — APPROVED** [review]
thanks sarah!

**Pull request merged**